packages/three-vrm-core/.gitignore

```diff
@@ -0,0 +1 @@
+types/
```

Comment:
Temp git ignore. After https://github.com/pixiv/three-vrm/pull/1293 types will end up in the lib dir